recaf-ui/src/main/java/me/coley/recaf/ui/control/tree/OutlineTree.java

```diff
@@ -0,0 +1,144 @@
+package me.coley.recaf.ui.control.tree;
+
+import javafx.scene.control.TreeCell;
+import javafx.scene.control.TreeItem;
+import javafx.scene.control.TreeView;
+import me.coley.recaf.code.*;
+import me.coley.recaf.config.Configs;
+import me.coley.recaf.ui.behavior.ClassRepresentation;
+import me.coley.recaf.ui.behavior.Updatable;
+import me.coley.recaf.ui.context.ContextBuilder;
+import me.coley.recaf.ui.util.Icons;
+import me.coley.recaf.util.AccessFlag;
+import me.coley.recaf.util.EscapeUtil;
+import me.coley.recaf.util.StringUtil;
+import me.coley.recaf.util.Types;
+import me.coley.recaf.util.threading.FxThreadUtil;
+import org.objectweb.asm.Type;
+
+import java.util.Arrays;
+import java.util.stream.Collectors;
+
+import static me.coley.recaf.ui.pane.OutlinePane.showSynthetics;
+import static me.coley.recaf.ui.pane.OutlinePane.showTypes;
+
+/**
+ * Tree that represents the {@link MemberInfo} of a {@link CommonClassInfo}.
+ */
```

Comment:
Can add back `@author` from the class this was pulled from now that its in its own file.